Source/Models/Proof.swift

```diff
@@ -10,9 +10,24 @@ public struct Proof {
     
     let nodeHash: NodeHash
     let status: Status
+    let metadata: [String: Any]?
```

Comment:
why is this `metadata` optional? Some `metadata` is always coming through the response, no?